packages/lesswrong/components/common/TabNavigationMenu/GivingPortalMobileMenuItem.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { registerComponent, Components } from '../../../lib/vulcan-lib/components';
+import { TabNavigationItemProps } from './TabNavigationItem';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    color: theme.palette.givingPortal.rhsLink,
+    '&:hover': {
+      color: theme.palette.givingPortal.rhsLink,
+      filter: "brightness(0.6)"
+    },
+    [theme.breakpoints.up('md')]: {
+      display: 'none'
+    },
+  }
+})
+
+const GivingPortalMobileMenuItem = ({tab, onClick, classes}: TabNavigationItemProps) => {
```

Comment:
The main purpose of this special component is to make the link the right color and only show it on small screens (where the header link isn't visible)

Reply:
IIRC the TabNavigationMenu supports having different items on mobile. Not gonna help you with the color tho

Reply:
That would be reasonable, but I didn't see anything for that